ts/shielder-sdk-crypto-wasm-light/src/circuits/teeClient.ts

```diff
@@ -0,0 +1,50 @@
+import { verifyAttestation } from "@/nitro-attestation";
+import * as secp from "@noble/secp256k1";
+
+type TeePublicKeyResponse = {
+  // secp256k1 public key in hex format
+  public_key: string;
+  // base64-encoded AWS Nitro attestation document
+  attestation_document: string;
+};
+
+export class TeeClient {
+  provingServiceUrl: string | undefined;
+
+  provingServicePublicKey: secp.Point | undefined;
+
+  async init(
+    provingServiceUrl: string,
+    withoutAttestation: boolean
+  ): Promise<void> {
+    this.provingServiceUrl = provingServiceUrl;
+
+    const response = await fetch(`${this.provingServiceUrl}/public_key`, {
+      method: "POST",
```

Comment:
`/public_key` is implemented as GET in `shielder-prover-server`, I assumed GET is enough here since we don't pass any parameters.